docs/alma/alma.rst

```diff
@@ -124,47 +123,37 @@ Querying Targets and Regions
 You can query by object name or by circular region:
 
 .. code-block:: python
-
+.. doctest-remote-data::
     >>> from astroquery.alma import Alma
     >>> m83_data = Alma.query_object('M83')
     >>> print(len(m83_data))
-    830
+    352
```

Comment:
hmm, right now I get 400 for this, so I think we should probably drop this print altogether. If nothing has returned then the tests will fail with the next line anyway